site/docs/v1/tech/apis/_application-oauth-configuration-response-body.adoc

```diff
@@ -46,8 +46,11 @@ The logout URL for the Application. FusionAuth will redirect to this URL after t
 [field]#application.oauthConfiguration.requireClientAuthentication# [type]#[Boolean]# [since]#Available since 1.3.0#::
 Determines if the OAuth 2.0 Token endpoint requires client authentication. If this is enabled, the client must provide client credentials when using the Token endpoint. The `client_id` and `client_secret` may be provided using a Basic Authorization HTTP header, or by sending these parameters in the request body using POST data.
 
+[field]#application.oauthConfiguration.requireRegistration# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.28.0#::
```

Comment:
> This configuration does not affect any other grant, and does not affect the API usage.

Reply:
The response objects don't need the default or optional flags.

Reply:
Good catch.  I gotta get better about copy-n-paste.